src/AspNetCoreModuleV2/InProcessRequestHandler/inprocess/inprocessapplication.cpp

```diff
@@ -845,7 +867,7 @@ IN_PROCESS_APPLICATION::ExecuteApplication(
     // set the callbacks
     s_Application = this;
 
-    hr = RunDotnetApplication(m_pConfig->QueryHostFxrArgCount(), m_pConfig->QueryHostFxrArguments(), pProc);
+    hr = RunDotnetApplication(hostfxrArgc, hostfxrArgv, pProc);
```

Comment:
Argc and Argv are never set. Use hostfxr options.

Reply:
That's the point - to avoid resolving options for callback case.

Reply:
In the non-callback case, we are calling dotnet will a NULL buffer. This codepath is still shared between both.

Reply:
Right